premerge/monitoring.md

```diff
@@ -0,0 +1,20 @@
+# Monitoring
+
+Presubmit monitoring is provided by Grafana.
+The dashboard link is [https://llvm.grafana.net/dashboards](https://llvm.grafana.net/dashboards).
+
+Grafana pulls its data from 2 sources: the GCP Kubernetes cluster & GitHub.
+Grafana instance access is restricted, but there is a publicly visible dashboard:
+- [Public dashboard](https://llvm.grafana.net/public-dashboards/6a1c1969b6794e0a8ee5d494c72ce2cd)
```

Comment:
This links to the old dashboard. Could you update the link to the new LLVM dashboard rather than the CI Status Demo one?

Reply:
good catch.